### ISSUE TITLE
Accept Healthchecks Over "Plain" NATS

### ISSUE THREAD

**Issue by danielsdeleo** [chef/automate#1124]
## Description

The state of rust language support for NATS is that there are multiple libraries that each have incomplete feature sets. The one hab is currently using supports nats-streaming features but doesn't do TLS with self-signed certs. To address this, we want to send health check events via "plain" NATS messaging instead of nats-streaming. This will result in reduced delivery reliability for health check messages, but we believe this is acceptable because the overall behavior of the system is convergent. Specifically, each health check message includes the entire state of the service as understood by Automate, and by default these are sent every 30s. Automate has some limited tracking of state changes (the "time wizard"), but given that feature's intended use case and limited scope, the impact of some missed messages should be relatively small.

tl;dr: we want TLS more than we need the additional delivery reliability of nats-streaming.

## Definition of Done

* Hab 0.83.0 (using nats-streaming) continues to work.
* health check messages sent from generators via plain NATS messaging are accepted and stored by Automate, and appear in the UI
